hack/tools/release/notes/list_test.go

```diff
@@ -65,3 +65,134 @@ func Test_buildSetOfPRNumbers(t *testing.T) {
 		})
 	}
 }
+
+func Test_githubFromToPRLister_listPRs(t *testing.T) {
+	tests := []struct {
+		name    string
+		fields  *githubFromToPRLister
+		args    ref
+		wantErr bool
+	}{
+		{
+			name: "Successful PR Listing",
+			fields: &githubFromToPRLister{
+				client: &githubClient{
```

Comment:
Wondering if we should better use some mock client instead of reaching out to github on unit tests (rate-limiting could apply and make the test flaky)

Reply:
I agree with @chrischdi
In this case, to reach out to GitHub generates possibility about flaky. For example, when some PR piles the commit in a short period of time, we'll met the rate limit error. Then it's not problem even if we use mock client since we know the result at this time. In the future, using mock might not be problem.